README.md

```diff
@@ -152,20 +153,25 @@ Set a meaningful cluster name for your secured cluster in the `CLUSTER_NAME` she
 ```sh
 CLUSTER_NAME="my-secured-cluster"
 ```
-Then install stackrox-secured-cluster-services (with the init bundle you generated earlier) using this command:
+Then install stackrox-secured-cluster-services (with the init bundle you generated earlier).
+
+**Note:** When deploying stackrox-secured-cluster-services on a different cluster than the one where stackrox-central-services is deployed, you will also need to specify the endpoint (address and port number) of Central via `--set centralEndpoint=<endpoint_of_central_service>` command-line argument.
```

Comment:
I'm not convinced about the placement of this note, but I haven't figured a better approach. Ideas?

Reply:
Well, you could instead of the note introduce a variable name for this, say `CENTRAL_ENDPOINT`, and tell the user to change it if deploying on a different cluster. Something like this:

> Set endpoint of central the secured cluster services should communicate to. If you're deploying stackrox-secured-cluster-services on the same cluster as stackrox-central-services, leave it as shown, otherwise change the value to the endpoint through which Central is accessible.
> ```sh
> CENTRAL_ENDPOINT="central.stackrox.svc:443"
> ```

Then later

> ```sh
> helm upgrade [...]
> --set clusterName="$CLUSTER_NAME" \
> --set centralEndpoint="$CENTRAL_ENDPOINT"
> ```

Reply:
Good idea, I have implemented that. I was about to open a separate PR for it, since this one is starting to grow big with different changes, but kept it here still, I hope that's fine.